packages/client/hmi-client/src/components/workflow/tera-workflow.vue

```diff
@@ -175,7 +175,7 @@ import Button from 'primevue/button';
 import TeraToggleableInput from '@/components/widgets/tera-toggleable-input.vue';
 import ContextMenu from 'primevue/contextmenu';
 import * as workflowService from '@/services/workflow';
-import { OperatorImport, OperatorNodeSize, getNodeMenu } from '@/services/workflow';
+import { OperatorImport, OperatorNodeSize, getNodeMenu, appendInputPort } from '@/services/workflow';
```

Comment:
`getNodeMenu` and `appendInputPort` should be accessed by `workflowService` for consistency

Reply:
done